README.md

```diff
@@ -5,23 +5,27 @@ Visit http://webpy.org/ for more information.
 [![build status](https://secure.travis-ci.org/webpy/webpy.png?branch=master)](https://travis-ci.org/webpy/webpy)
 [![Codecov Test Coverage](https://codecov.io/gh/webpy/webpy/branch/master/graphs/badge.svg?style=flat)](https://codecov.io/gh/webpy/webpy)
 
-The latest stable release `0.51` supports both Python 2.7 and Python >= 3.5.
+The latest stable release `0.60` only supports supports Python >= 3.5.
 To install it, please run:
 
+```
+# For Python 3
+python3 -m pip install web.py==0.60
+```
+
+If you are using Python 2, please use the 0.51 version. Please note that
+this is the last version that supports Python 2. 
 ```
 # For Python 2.7
 pip2 install web.py==0.51
```

Comment:
```suggestion
python3 -m install web.py<0.60
```
This should work even if we have to release another release for legacy Python.

Reply:
I didn’t use “python2 -m install” before ...

Reply:
I guess he meant `python2 -m pip install`

Reply:
I think it is better to mention the explicit verson. I'm going to change this to:

```
python2 -m pip install web.py==0.51
```